CHANGELOG

```diff
@@ -1,3 +1,6 @@
+To be released
+    - Add flex-shrink
+    - Update flex-intial
```

Comment:
Shouldn't this be the other way around? `flex-initial` (you got a typo there) was created, and `flex-shrink` was updated